libs/domain/cart/coupon/coupon.component.ts

```diff
@@ -67,7 +67,17 @@ export class CouponComponent extends CartComponentMixin(
           return html`
             <div>
               <oryx-icon .type="${IconTypes.Check}"></oryx-icon>
-              <span class="code">${coupon.code} </span>
+              <span class="code">
+                ${coupon.code}
+                <oryx-button
+                  .type=${ButtonType.Icon}
```

Comment:
need to add aria-label, the button without text inside does not pass accessibility check